AppCenterCrashes/AppCenterCrashesTests/MSACErrorLogFormatterTests.mm

```diff
@@ -473,6 +471,79 @@ - (void)testErrorLogFromCrashReportWithWrapper {
   [defaults stopMocking];
 }
 
+- (void)testErrorLogDeviceInfoFromPLCR {
+
+  // If
+  MSACMockUserDefaults *defaults = [MSACMockUserDefaults new];
+
+  // When
+  NSData *crashData = [MSACCrashesTestUtil dataOfFixtureCrashReportWithFileName:@"live_report_exception_marketing"];
+
+  // Then
+  XCTAssertNotNil(crashData);
+
+  // If
+  NSError *error = nil;
+  PLCrashReport *report = [[PLCrashReport alloc] initWithData:crashData error:&error];
+  MSACDevice *device = self.deviceMock;
+  device.appBuild = @"445";
+  device.appNamespace = @"com.microsoft.appcenter.native.ios.puppet";
+  device.model = @"x86_64";
+  device.osBuild = @"20e241";
+  device.osVersion = @"14.4";
+
+  // Properties copied from the SDKs device information
+  device.sdkName = @"appcenter.ios";
+  device.sdkVersion = @"4.1.0";
+  device.oemName = @"Apple";
+  device.osName = @"iOS";
+  device.timeZoneOffset = [NSNumber numberWithInt:400];
+  device.screenSize = @"1792x828";
+  device.carrierCountry = @"countryCode";
+  device.carrierName = @"carrierName";
+  device.locale = @"en";
+  device.wrapperSdkVersion = @"10.11.12";
+  device.wrapperSdkName = @"Wrapper SDK for iOS";
+  device.wrapperRuntimeVersion = @"13.14";
+  device.liveUpdateReleaseLabel = @"Release Label";
+  device.liveUpdateDeploymentKey = @"Deployment Key";
+  device.liveUpdatePackageHash = @"Package Hash";
+
+  // When
+  MSACAppleErrorLog *errorLog = [MSACErrorLogFormatter errorLogFromCrashReport:report];
+
+  // Then
+  XCTAssertNotNil(errorLog.device.appBuild);
```

Comment:
We are checking these values for notNil and notEqual to the original device field. But we don't check if the value equals to the value from crash log.
Based on the test name I would expect this test to verify if the device fields were set to exact values from the crash log.